test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1

```diff
@@ -0,0 +1,153 @@
+Describe "ConvertTo-Html Tests" -Tags "CI" {
+
+    BeforeAll {
+        $customObject = [pscustomobject]@{"Name" = "John Doe"; "Age" = 42; "Friends" = ("Jack", "Jill")}
+        $newLine = "`r`n"
+    }
+
+    function normalizeLineEnds([string]$text)
+    {
+        $text -replace "`r`n?|`n", "`r`n"
+    }
+
+    It "Test ConvertTo-Html with no parameters" {
+        $returnObject = $customObject | ConvertTo-Html
+        $returnObject -is [System.Array] | Should Be $true
```

Comment:
`Should Be $true` is an anti-pattern - the logs aren't very useful so try to avoid that.

In this case, you can use 'Should BeOfType`. See the list of Pester assertions [here](https://github.com/pester/Pester/wiki/Should).

And I'll be honest - I'm surprised it's an array - I would have expected a single string, so from that point of view, it might be worth a test to at least make it obvious if someone were to implement my expected behavior.